### DISCUSSION TITLE
Discusiones sobre ciencia de datos!

### DISCUSSION THREAD

**Discussion by Julenasti** [DatSciR/ciencia_datos#2] [Announcements]
✏️ Aquí podéis plantear dudas, sugerencias o compartir nuevos conocimientos relacionados con la ciencia de datos que consideréis oportuno compartir

**Comment by ion1241** [comment]
https://discord.gg/6Ajsbcja Acceso a Discord de R

**Comment by ion1241** [comment]
https://medialab.github.io/iwanthue/ Generador de paletas de colores

**Reply by Julenasti** [reply]
eskerrik asko! parece super útil!

**Comment by ion1241** [comment]
Copilot puede leer/aprender de los repositorios de Github al ser todo de Microsoft y, por lo tanto, lee y aprende codigo R. En RStudio, existe la posibilidad de habilitar Copilot de GitHub en RStudio para ayudarnos a escribir codigo, pero tiene truco: solo es gratis para estudiantes y profesores. Si lo activamos, tambien tendremos copilot en la web de Github disponible para preguntar, con funciones bastante cómodas, como que te explique o resuma archivos .R o .Rmd. 

How to access GitHub Student and free Copilot - Como acceder a Github Student y Copilot gratuito:
https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279
RStudio's latest version Guide on Copilot - Guia de uso de Copilot en RStudio para su ultima version :
https://docs.posit.co/ide/user/ide/guide/tools/copilot.html#using-copilot

**Comment by VeruGHub** [comment]
Dos herramientas que facilitan la exploración de bases de datos que he aprendido hoy:

- DataExplorer: https://cran.r-project.org/web/packages/DataExplorer/vignettes/dataexplorer-intro.html
- dlookr: https://choonghyunryu.github.io/dlookr/
